### DISCUSSION TITLE
Security JKS connection

### DISCUSSION THREAD

**Discussion by Fr0z3Nn** [provectus/kafka-ui#2117] [Q&A] [answered]
Hi, kafka-ui team,

Have your project possibility to connect to list of brokers using only jks file?
Info about kafka zookeeper is absent

**Comment by Haarolean** [comment, marked as answer]
Hi Sergey, I'm not sure I got the problem.
Is it about zookeeper or kafka brokers?
If there's a problem with zookeeper, I can tell you that we've dropped zookeeper support, so you don't have to bother with it.
